pkg/apis/triggers/v1beta1/event_listener_types.go

```diff
@@ -78,6 +78,7 @@ type CustomResource struct {
 type KubernetesResource struct {
 	Replicas           *int32             `json:"replicas,omitempty"`
 	ServiceType        corev1.ServiceType `json:"serviceType,omitempty"`
+	ServicePort        int32              `json:"servicePort,omitempty"`
```

Comment:
Minor/Consistency: Use *int32. Generally we use pointer types for nullable/optional fields so that it is easier to distinguish between explicit zero and not specified

Reply:
done